### PR TITLE
Fix UI Tests

### PR DESCRIPTION
**Pull request by mohadian**
## JIRA
[MOB-3188](https://karhoo.atlassian.net/browse/MOB-3188)

## DESCRIPTION
Fix for UI Tests

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [X] Bugfix 
- [ ] New feature 
- [X] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [ ] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [X] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [ ] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

File: uisdk/src/androidTest/java/com/karhoo/uisdk/trip/TripTests.kt [hunk 1]
```diff
@@ -333,7 +333,7 @@ class TripTests : Launch {
         mockTripSuccessResponse(
                 status = TRIP_STATUS_ARRIVED,
                 tracking = DRIVER_TRACKING,
-                details = TRIP_ARRIVED)
+                details = TRIP_DER_NO_VEHICLE_AND_DRIVER_DETAILS)
```
Comment (jsm20) [inline uisdk/src/androidTest/java/com/karhoo/uisdk/trip/TripTests.kt:336]: Is it the test that is wrong or the code for this? Should the user be able to cancel once the driver has arrived?

File: uisdk/src/androidTest/java/com/karhoo/uisdk/booking/braintree/BraintreeBookingTests.kt [hunk 2]
```diff
@@ -107,7 +107,8 @@ class BraintreeBookingTests : Launch {
         booking(this, null) {
             mediumSleep()
         } result {
-            checkSnackbarWithText(R.string.K0001)
+            // R.string.K0001
```
Comment (jsm20) [inline uisdk/src/androidTest/java/com/karhoo/uisdk/booking/braintree/BraintreeBookingTests.kt:110]: Does the comment need to be there? Same with the other test for K0001

Comment (mohadian) [reply uisdk/src/androidTest/java/com/karhoo/uisdk/booking/braintree/BraintreeBookingTests.kt:110]: I thought to add the resource ID to help the reader understand where the message come from

File: uisdk/src/androidTest/java/com/karhoo/uisdk/trip/TripTests.kt (hunk 1 above)
Comment (mohadian) [reply uisdk/src/androidTest/java/com/karhoo/uisdk/trip/TripTests.kt:336]: This is the test which is covering the latest changes. If I am not mistaken when the Trip status changes to the TRIP_ARRIVED we hide the Cancel button in this screen. therefore the test is in valid and wrong especially lines [348](https://github.com/karhoo/karhoo-android-ui-sdk/pull/104/files#diff-eafb9a51edd057d51a24ed370cee7a471c653859165d3e3698d800ff3648464fR348) and [349](https://github.com/karhoo/karhoo-android-ui-sdk/pull/104/files#diff-eafb9a51edd057d51a24ed370cee7a471c653859165d3e3698d800ff3648464fR349)

File: uisdk/src/androidTest/java/com/karhoo/uisdk/booking/braintree/BraintreeBookingTests.kt (hunk 2 above)
Comment (mohadian) [reply uisdk/src/androidTest/java/com/karhoo/uisdk/booking/braintree/BraintreeBookingTests.kt:110]: I can remove it if you think it is unnecessary.

**Review by jsm20 — APPROVED** [review]

**Pull request merged**